glass/lensing.py

```diff
@@ -224,6 +224,79 @@ def w(zi, zj, zk):
         delta12 = delta23
 
 
+@generator('zmin, zmax, delta, kappa -> kappa')
+def ia_nla(cosmo, A_ia, eta=0., z0=0., lbar=0., l0=1e-9, beta=0.):
+    r'''Implementation of intrinsic alignments using the NLA model
+
```

Comment:
Missing "parameters", "receives", and "yields" sections. For example, see [here](https://github.com/glass-dev/glass/blob/1e9eecf4add226bc7f2619e5cb06c321bec4ef74/glass/galaxies.py#L478-L493).